boa_tester/src/exec/mod.rs

```diff
@@ -57,17 +57,36 @@ impl TestSuite {
             println!();
         }
 
-        // Count passed tests
+        // Count passed tests and es specs
         let mut passed = 0;
         let mut ignored = 0;
         let mut panic = 0;
+        let mut es5_passed = 0;
+        let mut es6_passed = 0;
+        let mut es5_total = 0;
+        let mut es6_total = 0;
```

Comment:
Seeing that this is the same as the normal counters, maybe it would be good to extract this to its own `Statistics` struct with fields `total: u32, passed: u32, ignored: u32, panic: u32`. That way we can just have a nice `stats: Statistics, es5_stats: Statistics, es6_stats: Statistics` for `SuiteResult`.